Workbooks/WorkbooksMetadata.json

```diff
@@ -407,10 +407,10 @@
     "logoFileName": "trendmicro_logo.svg",
     "description": "Visualize and gain insights into the MITRE ATT&CK related activity detected by Trend Micro Deep Security.",
     "dataTypesDependencies": [ "TrendMicroDeepSecurity" ],
-    "dataConnectorsDependencies": [ "TrendMicro" ],
+    "dataConnectorsDependencies": [ "TrendMicroDeepSecurity" ],
```

Comment:
This change is not needed since the data connector ID won't change (refer to the comment on the if for data connector).

Reply:
Change reverted.